cmd/buildah/config.go

```diff
@@ -18,42 +18,50 @@ const (
 
 var (
 	configFlags = []cli.Flag{
+		cli.StringSliceFlag{
+			Name:  "annotation, a",
+			Usage: "`annotation` e.g. annotation=value, for the target image",
```

Comment:
Nit: would make more sense as `"add annotation e.g. annotation=value to target image"`

Reply:
Add "add" or "set" on each one of these.